api/test/test_play_asset_pack_manager.gd

```diff
@@ -161,3 +162,117 @@ func test_cancel_asset_pack_request_non_existing_pack_name():
+	
+	# join instantiated thread
+	mock_plugin._show_confirmation_thread.wait_to_finish()
+
+func show_cellular_data_confirmation_success_helper(status : bool, result : int):
+	# assert using callback, simulating the workflow of connecting callback to signal
+	assert_true(status)
+	assert_eq(result, PlayAssetPackManager.AssetPackStorageMethod.STORAGE_FILES)
+
+func test_show_cellular_data_confirmation_error():
+	var mock_plugin = FakeAndroidPlugin.new()
+	
+	# configure what should be emitted upon show_cellular_data_confirmation() call
+	mock_plugin.set_show_confirmation_response(false, 0, create_mock_asset_pack_java_lang_exception_dict())
+	var test_object = create_play_asset_pack_manager(mock_plugin)
+	
+	var request_object = test_object.show_cellular_data_confirmation()
+	
+	# connect to helper function, simulating the workflow of connecting callback to signal
+	request_object.connect("request_completed", self, "show_cellular_data_confirmation_error_helper")
+	
+	# yield to the request_completed signal for no longer than 1 seconds and assert for signal emitted
+	yield(yield_to(request_object, "request_completed", 1), YIELD)
+	assert_signal_emitted(request_object, "request_completed", "signal should have emitted")
+	
+	# assert using getters, simulating the workflow of yielding the signals
+	assert_true(not request_object.get_status())
+	assert_asset_pack_exception_eq_dict(request_object.get_error(), \
+		create_mock_asset_pack_java_lang_exception_dict())
+	
+	# join instantiated thread
+	mock_plugin._show_confirmation_thread.wait_to_finish()
+
+func show_cellular_data_confirmation_error_helper(status : bool, error : PlayAssetPackException):
+	# assert using callback, simulating the workflow of connecting callback to signal
+	assert_true(not status)
+	assert_asset_pack_exception_eq_dict(error, create_mock_asset_pack_java_lang_exception_dict())
+
+func test_remove_pack_success():
+	var mock_plugin = FakeAndroidPlugin.new()
+
+	# configure what should be emitted upon remove_pack() call
+	mock_plugin.set_remove_pack_response(true, {})
+	var test_object = create_play_asset_pack_manager(mock_plugin)
+	
+	var request_object = test_object.remove_pack("packName")
+	
+	# connect to helper function, simulating the workflow of connecting callback to signal
+	request_object.connect("request_completed", self, "remove_pack_success_helper")
+	
+	# yield to the request_completed signal for no longer than 1 seconds and assert for signal emitted
+	yield(yield_to(request_object, "request_completed", 1), YIELD)
+	assert_signal_emitted(request_object, "request_completed", "signal should have emitted")
+	
+	# assert using getters, simulating the workflow of yielding the signals
+	assert_true(request_object.get_status())
+	
+	# join instantiated thread
+	mock_plugin._remove_pack_thread.wait_to_finish()
+
+func remove_pack_success_helper(status : bool, error : PlayAssetPackException):
```

Comment:
Same as above, see if you can make this name more descriptive.

Reply:
Renamed to ```assert_remove_pack_is_success```